### ISSUE TITLE
(A11y severity 3) Improve lhnav action button accessibility

### ISSUE THREAD

**Issue by sathomas** [oaeproject/3akai-ux#3944]
The "Library" menu expands to show the "Share", "Add to folder" and "Delete" buttons when a user checks the checkbox for a document. An additional issue arises because these buttons are disabled until a checkbox is activated. Disabled form controls are not placed in the navigation flow (you can't Tab to them). This further increases the likelihood that users will be unable to find the buttons. This can be addressed by removing the HTML `disabled` attribute and using `aria-disabled=true/false` instead.
